python/ribasim/__init__.py

```diff
@@ -1,12 +1,14 @@
 __version__ = "0.1.1"
 
 
-from ribasim import utils
-from ribasim.basin import Basin
-from ribasim.edge import Edge
-from ribasim.fractional_flow import FractionalFlow
-from ribasim.level_control import LevelControl
-from ribasim.linear_level_connection import LinearLevelConnection
-from ribasim.model import Model
-from ribasim.node import Node
-from ribasim.tabulated_rating_curve import TabulatedRatingCurve
+from ribasim import utils as utils
+from ribasim.basin import Basin as Basin
```

Comment:
Did `black` add `as Basin`? I don't particularly mind, but I don't see why it would be preferred.

Reply:
Linters like mypy and ruff complain about unused symbols otherwise.

I saw this pattern in other projects like e.g. https://github.com/tiangolo/fastapi/blob/master/fastapi/__init__.py

Reply:
I see, thanks, so let's keep it as is.